### ISSUE TITLE
Failing clj-test on Erlang 21.3

### ISSUE THREAD

**Issue by oubiwann** [lfe/lfe#372]
```
  1) clj-tests:comp_test/0
     Failure/Error: ?assertEqual([48,46,53], (car (io_lib:format (126 46 49 102) (backquote ((comma asin-result))))))
       expected: "0.5"
            got: 48
     %% eunit_proc.erl:325:in `eunit_proc:with_timeout/3`
     Output: 
     Output: 
```

For full run, see: https://travis-ci.org/oubiwann/lfe/jobs/529378169 where you can note that it passes for all other previous versions ...

**Comment by oubiwann** [comment]
So it loooks like this is the result of a behavioural change in Erlang 21.0 ...in particular, see: https://bugs.erlang.org/browse/ERL-633

REPL session in Erlang 17.5 and 20.3:
```lisp
lfe> (funcall (clj:comp #'math:sin/1 #'math:asin/1) 0.5)                                 
0.49999999999999994
lfe> (io_lib:format "~.1f" `(,(funcall (clj:comp #'math:sin/1 #'math:asin/1) 0.5)))
("0.5")
lfe> (car (io_lib:format "~.1f" `(,(funcall (clj:comp #'math:sin/1 #'math:asin/1) 0.5))))
"0.5"
```
REPL session in Erlang 21.0 and 21.3:
```lisp
lfe> (funcall (clj:comp #'math:sin/1 #'math:asin/1) 0.5)
0.49999999999999994
lfe> (io_lib:format "~.1f" `(,(funcall (clj:comp #'math:sin/1 #'math:asin/1) 0.5)))
"0.5"
lfe> (car (io_lib:format "~.1f" `(,(funcall (clj:comp #'math:sin/1 #'math:asin/1) 0.5))))
48
```

**Comment by oubiwann** [comment]
Looks like a better choice might have been `(float_to_list ... '(#(decimal 1)))` anyway; updating to use that.

**Comment by oubiwann** [comment]
Fixed in 8b8c6a0. Will submit for review as part of #371.

**Comment by oubiwann** [comment]
Working now.
